example/twitter.z/contracts/Twitter.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.6.0;
+pragma solidity ^0.8.0;
```

Comment:
@Tunji17 we can probably bump up all example sites contracts to `^0.8.0` in this branch.

Reply:
okay